src/platforms/javascript/common/install/index.mdx

```diff
@@ -20,3 +20,35 @@ notSupported:
 ---
 
 <PageGrid />
+
+## When to use which Installation Method
+
+Depending on the concrete needs of your application, you may wonder which installation method you should use.
+
+### Why to use the Loader Script over NPM
+
+Using the Loader Script is the easiest way to start using Sentry. You only need to add a script tag to your application, and we'll take care of setting everything up correctly for you.
+
+- **Easy to set up**: You only need to [add a script tag](./loader/#using-the-loader) to your application.
+
+- **Easy to upgrade**: We'll take care of always shipping the latest version of the SDK to your users - no action needed on your side.
```

Comment:
Should we talk about _why_ that's important/valuable? It's so you're ALWAYS up-to-date with the latest feature releases, bug fixes, improvements, etc.!

Reply:
Plus one to that! It's always helpful to tell our users why they should do something :)

Reply:
Added this sentence:

> This ensures you'll always get the latest features and bug fixes of the Sentry SDK.